### DISCUSSION TITLE
Not getting battery info - Growatt SPH3600 - wrong record layout?

### DISCUSSION THREAD

**Discussion by KasperEdw** [johanmeijer/grott#148] [Q&A] [answered]
I have been trying to setup Grott and getting rid of my reliance on Growatt.

Grott is running and data is getting into Home Assistant, but I don't seem to get all data and I'm really missing battery information. 

System is Growatt SPH 3600 with smart meter

From grott.py I notice that it says:
" no matching record layout found, try generic"
 " Record layout used: T06NNNNX"

This seems wrong and it should use a newer record layout that supports battery.

How do I correct this?

Thanks, 
Kasper

**Comment by johanmeijer** [comment, marked as answer]
add invtype=sph in grott.ini [Generic] section (or use ginvtype="sph" environmental variable for docker)

**Reply by KasperEdw** [reply]
Worked like a charm!!
You are a lifesaver - thanks a lot!
